### PR TITLE
added alt text

### PR DESCRIPTION
**Pull request by v-emilypr**

### CODE REVIEW

File: iis/publish/using-web-deploy/powershell-scripts-for-automating-web-deploy-setup.md [hunk 1]
```diff
@@ -83,13 +83,13 @@ Enable publishing for an existing user on an existing site (the password will no
 
 [!code-console[Main](powershell-scripts-for-automating-web-deploy-setup/samples/sample1.cmd)]
 
-![](powershell-scripts-for-automating-web-deploy-setup/_static/image5.png)
+![Screenshot of Powershell console with results of scripting.](powershell-scripts-for-automating-web-deploy-setup/_static/image5.png)
```
Comment (MeeraDi) [inline iis/publish/using-web-deploy/powershell-scripts-for-automating-web-deploy-setup.md:86]: Screenshot of a Powershell console with results of scripting. (to be consistent with others)

File: iis/publish/using-web-deploy/powershell-scripts-for-automating-web-deploy-setup.md [hunk 2]
```diff
@@ -171,13 +171,13 @@ Create a database with user using the defaults:
 
 [!code-console[Main](powershell-scripts-for-automating-web-deploy-setup/samples/sample7.cmd)]
 
-![](powershell-scripts-for-automating-web-deploy-setup/_static/image16.png)
+![Screenshot of a Powershell console with scripting to create database.](powershell-scripts-for-automating-web-deploy-setup/_static/image16.png)
```
Comment (MeeraDi) [inline iis/publish/using-web-deploy/powershell-scripts-for-automating-web-deploy-setup.md:174]: "the" database or "a" database?

**Review by Rick-Anderson — APPROVED** [review]

**Comment by Rick-Anderson** [comment]
@v-emilypr check the build report
Line 73: [Warning] Missing alternate text for image 'add/_static/image8.png'. Alt text is required for accessibility.

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]
Fix missing atl from build report

**Comment by v-emilypr** [comment]
@Rick-Anderson Line 73 added and ready for final review.

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**